src/layouts/error.vue

```diff
@@ -1,305 +1,118 @@
+        case '4':
+          errorObj.image = pageNotFound;
+          errorObj.title = this.$t('error_page_not_found_title_404');
+          errorObj.description = this.$t(
+            'error_page_not_found_description_404'
+          );
+          break;
 
-    getOAuthRegisterAPI() {
-      const { from: qsID, referrer, utm_source: utmSource } = this.$route.query;
-      const { id: paramID } = this.$route.params;
-      return getOAuthRegisterAPI({
-        language: this.$i18n.locale,
-        from: qsID || paramID,
-        referrer,
-        utmSource,
-      });
-    },
-    i18nKeyBase() {
-      return `ERROR.${this.error.message}`;
-    },
-    i18nKeyTitle() {
-      return `${this.i18nKeyBase}.title`;
-    },
-    i18nKeyMessage() {
-      return `${this.i18nKeyBase}.message`;
-    },
-    isStringLocalizedError() {
-      return this.$te(this.i18nKeyBase, defaultLocale);
-    },
-    isObjectLocalizedError() {
-      return this.$te(this.i18nKeyMessage, defaultLocale);
-    },
-    isLoginError() {
-      return /^LOGIN_NEEDED.*/.test(this.error.message);
-    },
-    isLoginErrorFromCivicLikerRegistration() {
-      return this.isLoginError && /civic-register.*/.test(this.$route.name);
-    },
-    isCivicLikerRelatedError() {
-      return /^CIVIC_LIKER.*/.test(this.error.message);
-    },
-    formattedTitle() {
-      if (this.$te(this.i18nKeyTitle, defaultLocale)) {
-        return this.$t(this.i18nKeyTitle);
-      }
-      return undefined;
-    },
-    formattedMessage() {
-      if (this.isObjectLocalizedError) {
-        return this.$t(this.i18nKeyMessage);
-      }
-      if (this.isStringLocalizedError) {
-        return this.$t(this.i18nKeyBase);
-      }
-      if (this.isLoginError) {
-        return this.$t('ERROR.LOGIN_NEEDED');
-      }
+        case '5':
+          errorObj.image = severError;
+          errorObj.title = this.$t('error_page_not_found_title_500');
+          errorObj.description = this.$t(
+            'error_page_not_found_description_500'
+          );
+          break;
 
-      const { statusCode, message } = this.error;
-      return `[${statusCode}] ${message}`;
-    },
-  },
-  async mounted() {
-    // HACK: to handle firebase function double decoding iscn id, causing route to fail
-    if (this.$route.path.startsWith('/nft/iscn/iscn:/likecoin-chain')) {
-      this.$router.replace({
-        name: 'nft-iscn-iscnId',
-        params: {
-          iscnId: this.$route.path.replace('/nft/iscn/iscn:/', 'iscn://'),
-        },
-      });
-      return;
-    }
-
-    if (this.isLoginError) {
-      if (window.localStorage) {
-        const { name, params, query, hash } = this.$route;
-        window.localStorage.setItem(
-          'USER_POST_AUTH_ROUTE',
-          JSON.stringify({
-            route: {
-              name,
-              params,
-              query,
-              hash,
-            },
-            ts: Date.now(),
-          })
-        );
+        default:
```

Comment:
Seem default same as case 4?
```suggestion
        case '4':
        default:
```

Reply:
oh!